source/documentation/syntax/special-functions.liquid

```diff
@@ -0,0 +1,156 @@
+---
+title: Special Functions
+introduction: >
+  CSS defines many functions, and most of them work just fine with Sass’s normal
+  function syntax. They’re parsed as function calls, resolved to [plain CSS
+  functions](../at-rules/function#plain-css-functions), and compiled as-is to
+  CSS. There are a few exceptions, though, which have special syntax that can’t
+  just be parsed as a [SassScript expression](structure#expressions). All
+  special function calls return [unquoted strings](../values/strings#unquoted).
+table_of_contents: true
+complementary_content: |
+  <nav aria-labelledby="page-sections" class="page-sections sl-c-list-navigation-wrapper sl-c-list-navigation-wrapper--collapsible">
+
+  <h3 id="page-sections">Page Sections</h3>
+
+  - [<code>url()</code>](#url())
+  - [<code>element(),</code> <code>progid:...(),</code> and <code>expression()</code>](#element()%2C-progid%3A...()%2C-and-expression())
+
+  </nav>
+---
+{% markdown %}
+## `url()`
+
+The [`url()` function][] is commonly used in CSS, but its syntax is different
+than other functions: it can take either a quoted _or_ unquoted URL. Because an
+unquoted URL isn't a valid SassScript expression, Sass needs special logic to
+parse it.
+
+[`url()` function]: https://developer.mozilla.org/en-US/docs/Web/CSS/url
+
+If the `url()`'s argument is a valid unquoted URL, Sass parses it as-is,
+although [interpolation][] may also be used to inject SassScript values. If it's
+not a valid unquoted URL—for example, if it contains [variables][] or [function
+calls][]—it's parsed as a normal [plain CSS function call][].
+
+[interpolation]: ../interpolation
+[variables]: ../variables
+[function calls]: ../at-rules/function
+[plain CSS function call]: ../at-rules/function#plain-css-functions
+
+{% endmarkdown %}
+
+{% codeExample 'url' %}
+$roboto-font-path: "../fonts/roboto";
+
+@font-face {
+// This is parsed as a normal function call that takes a quoted string.
+src: url("#{$roboto-font-path}/Roboto-Thin.woff2") format("woff2");
+
+font-family: "Roboto";
+font-weight: 100;
+}
+
+@font-face {
+// This is parsed as a normal function call that takes an arithmetic
+// expression.
+src: url($roboto-font-path + "/Roboto-Light.woff2") format("woff2");
+
+font-family: "Roboto";
+font-weight: 300;
+}
+
+@font-face {
+// This is parsed as an interpolated special function.
+src: url(#{$roboto-font-path}/Roboto-Regular.woff2) format("woff2");
+
+font-family: "Roboto";
+font-weight: 400;
+}
+===
+$roboto-font-path: "../fonts/roboto"
```

Comment:
I noticed the code on this page is not indented and I believe we want it to be to match what is shown on the Style Rules pages in this PR